### DISCUSSION TITLE
Vibration Sensor Support

### DISCUSSION THREAD

**Discussion by Codkit** [RavenSystem/esp-homekit-devices#1497] [Q&A] [answered]
Can i request support of vibration sensor please ? Thank you

**Comment by RavenSystem** [comment, marked as answer]
Sorry, vibration sensor is not supported by Apple. Apple HomeKit services (those that work with Home App) are defined by Apple and can not be modified by others. You can use a physical vibration sensor, but then you will must use a sensor from this list: https://github.com/RavenSystem/esp-homekit-devices/wiki/Sensors

**Comment by peros550** [comment]
Aqara sells a vibration sensor that can also be exposed in HomeKit as a motion sensor. That’s a workaround of course if anybody is interested.
